drivers/lte_link_control/Kconfig

```diff
@@ -52,6 +52,12 @@ config LTE_LOCK_PLMN_STRING
 		Only numeric string formats supported.
 endif # LTE_LOCK_PLMN
 
+config LTE_UNLOCK_PLMN
```

Comment:
This option should depend on `!LTE_LOCK_PLMN`, otherwise both these options could be selected simultaneously (resulting in the unlocking option being simply ignored without any warning, what may bring some confusion).

Reply:
Thank you, updated